flit/wheel.py

```diff
@@ -155,6 +156,9 @@ def write_metadata(self):
         elif self.ini_info['entry_points_file'] is not None:
             self._add_file(self.ini_info['entry_points_file'],
                            dist_info + '/entry_points.txt')
+        for base in ('COPYING', 'LICENSE'):
+            for fname in sorted(self.directory.glob(base + '*')):
+                self._add_file(fname, '%s/%s' % (dist_info, fname))
```

Comment:
If the directory is not the cwd, its path will be added between dist_info and the filename. We can avoid this by taking `fname.name` for the second formatting argument (that's equivalent to `os.path.basename()`).